features-support/step_definitions/features.js

```diff
@@ -2,14 +2,16 @@
 
 var should = require('should');
 var By = require('selenium-webdriver').By;
+var until = require('selenium-webdriver').until;
+var fs = require('fs-extra');
 
 const pageLoadTimeout = 30 * 1000;
 const timeoutObject = {timeout: pageLoadTimeout};
 
 // Deal with the non-standard webdriver promises.
```

Comment:
This function was only here because of the way WebDriver promises behaved, if they have been standardised it might be redundant now?